common/lib/xmodule/xmodule/js/main_requirejs.js

```diff
@@ -0,0 +1,8 @@
+(function(requirejs) {
+    requirejs.config({
+        paths: {
+            "moment": "xmodule/include/common_static/js/vendor/moment.min"
```

Comment:
I don't think there is any requirejs config for tests in common. Like I mentioned, the common JS depends on the parent location, so the video code doesn't declare dependencies on third-party libraries like jquery, underscore, etc.

I think that what is going on is that moment realizes that it is being loaded in a RequireJS environment, and therefore doesn't put moment on the window object. This may be due to changes we made as part of the LMS optimizer workflow, that now moment is behaving differently, even for our namespaced version of RequireJS.

An option that may work is to use the "init" option in the shim configuration of RequireJS (in LMS and CMS)-- inside init, put moment on the window object. Then you could keep the code in 01_initialize.js as-is, and the tests would also work as-is.